experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/PopulationSizing/index.tsx

```diff
@@ -0,0 +1,286 @@
+      </Form.Group>
+      <Form.Label
+        className="d-flex align-items-center"
+        data-testid="new-total-clients-label"
+      >
+        {totalNewClients} total{" "}
+        <span
+          style={{ fontWeight: 700, marginLeft: "0.3em", marginRight: "0.3em" }}
+        >
+          new
+        </span>{" "}
+        clients for given parameters
+      </Form.Label>
+      <Form.Label
+        className="d-flex align-items-center"
+        data-testid="existing-total-clients-label"
+      >
+        {totalExistingClients} total{" "}
+        <span
+          style={{ fontWeight: 700, marginLeft: "0.3em", marginRight: "0.3em" }}
+        >
+          existing
+        </span>{" "}
+        clients for given parameters
+      </Form.Label>
+      <p>
+        {Object.keys(newUserSizing.sample_sizes).map((powerKey) => {
+          // powerKey is the same for new and existing so we can look both up with the same key
+          const newUserMetrics =
+            newUserSizing.sample_sizes[powerKey]["metrics"];
+          const existingUserMetrics =
+            existingUserSizing.sample_sizes[powerKey]["metrics"];
+          // parameters are the same for new and existing so we only need to get one
+          const parameters = newUserSizing.sample_sizes[powerKey]["parameters"];
+          return (
+            <>
+              <Form.Label style={{ fontWeight: 700 }}>
+                Effect Size: {parameters.effect_size * 100}%
+              </Form.Label>
+              <table className="table-visualization-center border">
+                <thead>
+                  <tr>
+                    <th scope="col" className="border-bottom-0 bg-light" />
+                    {Object.keys(newUserMetrics).map((metricKey) => (
+                      <th
+                        key={metricKey}
+                        className="border-bottom-0 bg-light"
+                        scope="col"
+                      >
+                        <div>{metricKey}</div>
+                      </th>
+                    ))}
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr key={`${powerKey}-metrics`}>
+                    <th className="align-middle" scope="row">
+                      New Clients
+                    </th>
+                    {Object.keys(newUserMetrics).map((metricKey) => (
+                      <td className="align-middle" key={`${metricKey}-sizing`}>
+                        <table
+                          className="table-visualization-center-no-border"
+                          style={{ padding: 0, margin: 0 }}
+                        >
+                          <tbody>
+                            <tr>
+                              <td
+                                style={{
+                                  textAlign: "end",
+                                  border: "none",
+                                  padding: "0.25rem",
+                                }}
```

Comment:
Yeah try to avoid explicitly declaring element styles and use the bootstrap classes where ever possible 🙏